modules/ROOT/pages/external-configuration.adoc

```diff
@@ -40,15 +40,15 @@ Thus, it is important to change the configuration of microservices without updat
 
 == MicroProfile Config
 
-xref:reference:feature/mpConfig-1.4.adoc[MicroProfile Config] separates the configuration from code.
+xref:mpConfig-1.4.adoc[MicroProfile Config] separates the configuration from code.
```

Comment:
use the macro so the doc will always resolve to the most recent feature version
`feature:mpConfig[display=MicroProfile  Config]`